Assignment1/ViewController.swift

```diff
@@ -22,7 +22,7 @@ class ViewController: UIViewController {
         // Do any additional setup after loading the view.
     }
     
-    @IBAction func TappedSaveButton(_ sender: Any) {
+    @IBAction func SaveButtonPressed(_ sender: Any) {
```

Comment:
Have a look at function naming convention. Swift docs suggest the following, "Follow case conventions. Names of types and protocols are UpperCamelCase. Everything else is lowerCamelCase."

https://swift.org/documentation/api-design-guidelines/